CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,103 @@
+  or threatened is not a productive one. Members of the Coq development team
+  and user community should be respectful when dealing with other members as
+  well as with people outside the community.
+- **Be careful in the words that you choose.**
+  We are a community of professionals, and we conduct ourselves professionally.
+  Be kind to others. Do not insult or put down other participants. Harassment
+  and other exclusionary behavior aren't acceptable. This includes, but is not
+  limited to:
+  * Violent threats or language directed against another person.
+  * Discriminatory jokes and language.
+  * Posting sexually explicit or violent material.
+  * Posting (or threatening to post) other people's personally identifying
+    information ("doxing").
+  * Personal insults, especially those using racist or sexist terms.
+  * Unwelcome sexual attention.
+  * Advocating for, or encouraging, any of the above behavior.
+  * Repeated harassment of others. In general, if someone asks you to stop,
+    then stop.
+- **Remember that what you write in a public online forum might be read by
+  many people you don't know.**
+  Consider what image your words will give to outsiders of the development
+  team / the user community as a whole. Try to avoid references to private
+  knowledge to be understandable by anyone.
+- **When we disagree, try to understand why.**
+  Disagreements, both social and technical, happen all the time and Coq is no
+  exception. It is important that we resolve disagreements and differing views
+  constructively. Remember that we're different. There is strength in a varied
+  community, with people from a wide range of backgrounds. Different people
+  have different perspectives on issues. Being unable to understand why someone
+  holds a viewpoint doesn't mean that they're wrong. Don't forget that it is
+  human to err and blaming each other doesn't get us anywhere. Instead, focus
+  on helping to resolve issues and learning from mistakes.
+
+## Enforcement ##
+
+If you believe someone is violating the code of conduct, we ask that you report
+it by emailing the Coq Code of Conduct enforcement team at
+<coq-conduct@inria.fr>. Confidentiality with regard to the reporter of an
+incident will be maintained while dealing with it.
+
+In particular, you should seek support from the team instead of dealing by
+yourself with a behavior that you consider hurtful.
+
+Depending on the violation, the team can choose to address a private or public
+warning to the offender, request an apology, or ban them for a short or a long
+period of time from interacting on one or all of our forums.
+
+## Questions? ##
+
+If you have questions, feel free to write to <coq-conduct@inria.fr>.
+
+## Attribution ##
+
+This text is adapted from the
+[Django Code of Conduct](https://www.djangoproject.com/conduct/)
+which itself was adapted from the Speak Up! Community Code of Conduct.
```

Comment:
The django coc links to an archive.org version of the speakup one because speakup.io disappeared. Should we link to an archive.org version of django in anticipation of them disappearing?
It would also be more accurate as to what we started with if they change their code of conduct.

Reply:
Why not?